src/components/language-switch/language-switch.tsx

```diff
@@ -18,26 +23,30 @@ export const LanguageSwitch = ({
   translation,
   className = 'language-switch',
 }) => {
-  const { languages, language, originalPath } = useI18next();
+  const { languages, language, originalPath, t } = useI18next();
+  const languageSwitchLabel = t('navigation.language-aria');
 
   return (
-    <div className={className}>
-      {languages.map((languageOfLink) => {
-        return (
-          <StyledLanguageLink
-            key={languageOfLink}
-            to={
-              language !== languageOfLink && translation
-                ? translation
-                : originalPath
-            }
-            language={languageOfLink}
-            selected={language === languageOfLink}
-          >
-            {languageOfLink}
-          </StyledLanguageLink>
-        );
-      })}
-    </div>
+    <nav aria-label={languageSwitchLabel} className={className}>
+      <ul>
+        {languages.map((languageOfLink) => {
+          return (
+            <UnstyledListItem key={languageOfLink}>
+              <StyledLanguageLink
+                to={
+                  language !== languageOfLink && translation
+                    ? translation
+                    : originalPath
+                }
+                language={languageOfLink}
+                selected={language === languageOfLink}
+              >
+                {languageOfLink}
```

Comment:
The problem could be the `de` and `en`. I think we should provide at least a title with the actual language name. This will also help accessibility to spell the language instead of having some cryptic characters?

Something n`< a title="English" .. >`. Can you add this ?

Reply:
Yep, but in a separate PR